resources/zib/zib-DevelopmentChild.xml

```diff
@@ -0,0 +1,325 @@
+      <path value="Observation.component.value[x]" />
+      <short value="ToiletTrainednessUrine" />
+      <definition value="Degree of toilet trainedness for urine." />
+      <alias value="ZindelijkheidUrine" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.32.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-developmentchild-v1.2-2020EN" />
+        <map value="NL-CM:4.32.4" />
+        <comment value="ToiletTrainednessUrine" />
+      </mapping>
+    </element>
+    <element id="Observation.component:toiletTrainednessFeces">
+      <path value="Observation.component" />
+      <sliceName value="toiletTrainednessFeces" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:toiletTrainednessFeces.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="275818006" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:toiletTrainednessFeces.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="ToiletTrainednessFeces" />
+      <definition value="Degree of toilet trainedness for feces." />
+      <alias value="ZindelijkheidFeces" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.32.2--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-developmentchild-v1.2-2020EN" />
+        <map value="NL-CM:4.32.5" />
+        <comment value="ToiletTrainednessFeces" />
+      </mapping>
+    </element>
+    <element id="Observation.component:ageFirstMenstruation">
+      <path value="Observation.component" />
+      <sliceName value="ageFirstMenstruation" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:ageFirstMenstruation.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="42798-9" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:ageFirstMenstruation.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="AgeFirstMenstruation" />
+      <definition value="The age of the start of the first menstruation." />
+      <alias value="LeeftijdEersteMenstruatie" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <patternQuantity>
+        <system value="http://unitsofmeasure.org" />
+      </patternQuantity>
```

Comment:
I think the .code value with the UCUM unit for years (a) needs to be added here.

Reply:
Added ucum unit